content/docs/pipelines/pipelines-tutorial.md

```diff
@@ -0,0 +1,616 @@
+        class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Click **Pipeline Dashboard** to access the pipelines UI. The pipelines UI
+   looks like this:
+    <img src="/docs/images/pipelines-ui.png" 
+      alt="Pipelines UI"
+      class="mt-3 mb-3 border border-info rounded">
+
+Notes:
+
+* When the deployment has finished, you should have a running cluster in the 
+  cloud ready to run your code. You can interact with the cluster either by 
+  using [`kubectl`][kubectl] or by going to the
+  [GKE page on the GCP Console][gcp-console-kubernetes-engine].
+
+* While the deployment is running, you can watch your resources appear on the
+  GCP console:
+    * [Deployment on Deployment Manager][gcp-console-deployment-manager]
+    * [Cluster on GKE][gcp-console-kubernetes-engine]
+    * [Computing resources on Compute Engine][gcp-console-compute-engine]
+
+* It can take 10-15 minutes for the URI to become available. Kubeflow needs
+  to provision a signed SSL certificate and register a DNS name.
+    * If you own/manage the domain or a subdomain with [Cloud DNS][dns]
+      then you can configure this process to be much faster.
+    * While you wait you can access Kubeflow services by using `kubectl proxy`
+      and `kubectl port-forward` to connect to services in the cluster.
+
+### Create a Cloud Storage bucket
+
+The next step is to create a Cloud Storage bucket to hold your trained model
+
+[Cloud Storage][cloud-storage] is a scalable, fully-managed object/blob store.
+You can use it for a range of scenarios including serving website content,
+storing data for archival and disaster recovery, or distributing large data
+objects to users via direct download. This tutorial uses Cloud Storage to
+hold the trained machine learning model and associated data.
+
+Use the [`gsutil mb`][gsutil-mb] command to create a storage bucket. Your
+*bucket name* must be unique across all of Cloud Storage.
+The following commands create a bucket in the `us-central1` region,
+which corresponds to the `us-central1-c` zone used earlier
+in the tutorial:
+
+```
+export BUCKET_NAME=${PROJECT}-${DEPLOYMENT_NAME}-bucket
+gsutil mb -c regional -l us-central1 gs://${BUCKET_NAME}
+```
+
+## Prepare your pipeline
+
+To simplify this tutorial, you can use a set of prepared files that include
+the pipeline definition and supporting files. The project files are in the 
+[Kubeflow examples repository](https://github.com/kubeflow/examples/tree/master/pipelines/mnist-pipelines)
+on GitHub.
+
+### Download the project files
+
+Clone the project files and go to the directory containing the MNIST pipeline
+example:
+
+```
+cd ${HOME}
+git clone https://github.com/kubeflow/examples.git
+cd examples/pipelines/mnist-pipelines
+```
+
+As an alternative to cloning, you can download the 
+[Kubeflow examples repository zip file](https://github.com/kubeflow/examples/archive/master.zip).
+
+### Set up Python
+
+You need **Python 3.5 or above**. If you don't have Python 3 set up, install
```

Comment:
This might be just about Python versions available by OS/tool, but here you state that you need Python 3.5 or better, then you talk about installing Python 3, and in the code below you are using Python 3.7. Any way to clarify or add context around the different version references?

Reply:
Done.